master/docs/manual/cfg-workers-openstack.rst

```diff
@@ -162,6 +162,28 @@ When the instance is destroyed, the volume will be terminated as well.
     ]
 
 
+The ``nova_args`` can be used to specify additional arguments for the novaclient.
+For example network mappings, which is required if your OpenStack tenancy has more than one network, and default cannot be determined.
+Please refer to your OpenStack manual whether it wants net-id or net-name.
+
+Other useful parameters are availability_zone, security_groups and config_drive.
```

Comment:
I would format Python parameters as raw block:

```
Other useful parameters are ``availability_zone``, ``security_groups`` and ``config_drive``.
```

Honestly I would expect Sphinx to report them as spelling errors...